yarn-project/simulator/src/client/private_execution.test.ts

```diff
@@ -282,7 +288,7 @@ describe('Private Execution test suite', () => {
     const mockFirstNullifier = new Fr(1111);
     let currentNoteIndex = 0n;
 
-    const buildNote = (amount: bigint, owner: AztecAddress, storageSlot: Fr, noteTypeId: Fr) => {
+    const buildNote = (amount: bigint, ownerNpkMH: Fr, storageSlot: Fr, noteTypeId: Fr) => {
```

Comment:
Everywhere else you call the npkMHash and here you have just `npkMH`

Reply:
I still see `accountOrNpkMHash` all over the place so seems to not be complete
<img width="1011" alt="image" src="https://github.com/AztecProtocol/aztec-packages/assets/13470840/2c025537-6d36-45a7-997e-d689d2d4ee11">